packages/config/config/devices/0x0455/ora-zrx.json

```diff
@@ -0,0 +1,203 @@
+{
+	"manufacturer": "Cherubini S.p.A.",
+	"manufacturerId": "0x0455",
+	"label": "Ora ZRX",
+	"description": "Z-Wave Tubular Motor",
+	"devices": [
+		{
+			"productType": "0x0004",
+			"productId": "0x0001"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"description": "Lifeline group",
```

Comment:
Descriptions for association groups are unnecessary and unused by any UI. Please remove them.

Reply:
Done